### PR TITLE
docs(css): Follow-up PR for `shape()` function page

### PR DESCRIPTION
**Pull request by dipikabh**
### Description

This is a continuation of https://github.com/mdn/content/pull/33446 to address the remaining review comments.

### CODE REVIEW

**Comment by dipikabh** [comment]
Hi @estelle, @OnkarRuikar, your comments and suggestions are addressed in this PR. PTAL and let me know if these changes look okay to land. Thanks!

File: files/en-us/web/css/basic-shape/shape/index.md [hunk 1]
```diff
@@ -228,6 +256,15 @@ body {
   margin: 20px;
   background-color: lightgray;
 }
+
+@supports not (offset-path: shape(from 0 0, move to 0 0)) {
```
Comment (OnkarRuikar) [inline files/en-us/web/css/basic-shape/shape/index.md:260]:
```suggestion
@supports not (clip-path: shape(from 0 0, move to 0 0)) {
```

File: files/en-us/web/css/basic-shape/shape/index.md [hunk 2]
```diff
@@ -169,19 +190,26 @@ body {
   position: absolute;
   text-align: center;
   line-height: 50px;
-  animation: move 5s infinite alternate ease-in-out;
+  animation: move 6s infinite ease-in-out;
```
Comment (OnkarRuikar) [inline files/en-us/web/css/basic-shape/shape/index.md:193]:
```suggestion
  animation: move 6s infinite linear;
```
It's easier to understand what's going one with `linear`;

File: files/en-us/web/css/basic-shape/shape/index.md [hunk 3]
```diff
@@ -169,19 +190,26 @@ body {
...
 }
 
 .shape1 {
-  offset-path: shape(from 30% 60px, curve to 180px 180px via 90px 190px);
+  offset-path: shape(
+    from 30% 60px,
+    curve to 180px 180px via 90px 190px,
+    close,
+    move by 50px 30px,
+    hline by 30%
```
Comment (OnkarRuikar) [inline files/en-us/web/css/basic-shape/shape/index.md:202]:
```suggestion
    move by 0px 150px,
    hline by 40%
```
The `move by` effect is not noticeable. Let's exaggerate it.

**Review by OnkarRuikar — COMMENTED** [review]
A few more nits.\
Due to jumps in paths it's not clear where the animations start. To indicate animation progress could you do following change:
```diff
@keyframes move {
  0% {
    offset-distance: 0%;
  }
  100% {
    offset-distance: 100%;
++    background-color: pink;
  }
}
```

File: files/en-us/web/css/basic-shape/shape/index.md (hunk 1 above)
Comment (dipikabh) [reply files/en-us/web/css/basic-shape/shape/index.md:260]: Good catch :) Thanks!

**Comment by dipikabh** [comment]
> Due to jumps in paths it's not clear where the animations start. 

Fixed in https://github.com/mdn/content/pull/33783/commits/8380d65b6bab35abf71e32656592a39cc38efcde

**Review by OnkarRuikar — COMMENTED** [review]
Looks good to me.

**Comment by dipikabh** [comment]
Thanks, @OnkarRuikar 👍

File: files/en-us/web/css/basic-shape/index.md [hunk 4]
```diff
@@ -107,7 +107,7 @@ The {{cssxref("basic-shape/path","path()")}} function defines a shape using an S
 path( <`fill-rule`>?, ]? <string> )
 ```
 
-The required `<string>` is an [SVG Path](/en-US/docs/Web/SVG/Attribute/d) as a quoted string.
+The required `<string>` is an [SVG Path](/en-US/docs/Web/SVG/Attribute/d) as a quoted string. The `path()` function is not a valid {{cssxref("shape-outside")}} property value.
```
Comment (pepelsbey) [inline files/en-us/web/css/basic-shape/index.md:110]:
```suggestion
The required `<string>` is an [SVG path](/en-US/docs/Web/SVG/Attribute/d) as a quoted string. The `path()` function is not a valid {{cssxref("shape-outside")}} property value.
```

**Review by pepelsbey — APPROVED** [review]
Just a small suggestion. Otherwise, looks good! Thanks :)

**Pull request merged**